.github/workflows/CI.yml

```
@@ -182,3 +182,66 @@ jobs:
         with:
           key: ${{ steps.build-cache.outputs.cache-primary-key }}
           path: ~/.cache
+  release:
+    name: "Publish Pre-Release"
+    runs-on: ubuntu-20.04-16core
```

Don't need the big machine for this
```suggestion
    runs-on: ubuntu-latest
```